client/components/Treemap.jsx

```diff
@@ -91,6 +91,10 @@ export default class Treemap extends Component {
       },
       onGroupClick(event) {
         preventDefault(event);
+        if ((event.ctrlKey || event.secondary) && props.onGroupSecondaryClick) {
```

Comment:
Re: `event.metaKey`, Chrome and Firefox in OSX trigger the native context menu only when `e.ctrlKey === true`, not `e.metaKey === true`. But I should test in more browsers/OSes again and also don't see a downside to adding an extra trigger key, it could make sense to add that either way.

Also I can document that this is a FoamTree event because that's less-than-obvious -- would you be averse to using a JsDoc-style comment for that?

Reply:
> would you be averse to using a JsDoc-style comment for that

That's up to you, but I think just a simple one-liner will work as well.